includes/Views/Checkout.php

```diff
@@ -89,8 +92,8 @@ class="button alt wp-element-button fa-search ffl-search-button"><?php echo __('
                                     <button type="button" id="automaticffl-search-button" value="12" class="button alt ffl-search-button"></button>
                                 </div>
                                 <div class="modal-header-search">
-                                    <p id="ffl-results-message"></p>
                                     <p id="ffl-searching-message"><?php echo __( 'Looking for dealers, please wait...' ); ?></p>
+                                    <p id="ffl-results-message"></p><span id="toggle-map-text" class="hidden show-text-map">View map</span>
```

Comment:
Always use the function __() when printing a string:

https://developer.wordpress.org/reference/functions/__/